consensus/spos/roundConsensus_test.go

```diff
@@ -4,7 +4,6 @@ import (
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go/consensus/spos"
-	"github.com/ElrondNetwork/elrond-go/consensus/spos/bn"
```

Comment:
I think this should also remain, but with bls not bn.

Reply:
done.

Reply:
ok